cozy/ui/book_detail_view.py

```diff
@@ -103,26 +107,37 @@ def _on_book_changed(self):
             return
 
         book = self._view_model.book
+
+        try:
+            self._chapters_thread.join(timeout=0.2)
```

Comment:
Note to self: using this instead or in addition to the join call above: https://stackoverflow.com/questions/323972/is-there-any-way-to-kill-a-thread/325528#325528